CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# How to contribute
+This projects welcomes contributions and corrections. Before contributing, please make sure you have read the guidelines below. 
+If you decide to contribute anything, please do the following:
+
+
+## General Instructions
```

Comment:
nit: can you un-capitalize "Instructions" (let's use a consistent format for all the titles)